proposals/nnnn-remote-control-onRcStatus-notification.md

```diff
@@ -0,0 +1,92 @@
+# Remote Control - OnRCStatus notification
+
+* Proposal: [SDL-NNNN](nnnn-remote-control-onRcStatus-notification.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+One simple resource allocation rule is that at any given time only one application controls a RC module. For example, if SDL allocates a climate control module to application ONE, SDL rejects other applications to change the settings of the same climate control module. The module is freed when application ONE exits, disconnects, goes to HMI level NONE or no longer has the right to control the module due to various reasons (for example, a policy update revokes the access right to the module from the app, or SDL re-allocates the module to another app). In the last case, the mobile application does not aware the change.
+
+
+A new notification originated from SDL is nice to have. If SDL allocates a module to an application or it deallocates a module from an application, SDL shall send OnRCStatus notifications to both the mobile application and the HMI. The notification shall contain an array of the RC modules allocated to the application, so that the mobile application can know it gains or loses control of a module before a failure request, and the HMI can update the status on which application controls which modules.
```

Comment:
Please change "deallocates" to "de-allocates" to match formatting throughout proposal.

Reply:
changed, thanks